### PR TITLE
CB-9879 Allow option to not start datahub cluster automatically when …

### PR DESCRIPTION
**Pull request by doktoric**
…environment starts up

See detailed description in the commit message.

### CODE REVIEW

**Comment by doktoric** [comment]
retest this please

**Review by topolyai5 — CHANGES_REQUESTED** [review]
I have a suggestion. If we plan to add the `Recover to before stop` state, much better to use an enum with 2 different states. We can extend the num easily

File: environment/src/main/java/com/sequenceiq/environment/environment/dto/EnvironmentStartDto.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
+package com.sequenceiq.environment.environment.dto;
```
Comment (topolyai5) [inline environment/src/main/java/com/sequenceiq/environment/environment/dto/EnvironmentStartDto.java:1]: is this a good package? I think it does not relate to telemetruy

File: common-model/src/main/java/com/sequenceiq/common/api/telemetry/model/DataHubStartAction.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package com.sequenceiq.common.api.telemetry.model;
```
Comment (topolyai5) [inline common-model/src/main/java/com/sequenceiq/common/api/telemetry/model/DataHubStartAction.java:1]: is this a good package? I think it does not relate to telemetruy

**Comment by doktoric** [comment]
restest this please

**Review by topolyai5 — APPROVED** [review]

**Comment by doktoric** [comment]
retest this please

**Pull request merged**